src/components/Footer/footerData/siteMap.js

```diff
@@ -0,0 +1,6 @@
+export const SITEMAP = [
```

Comment:
```suggestion
export const SITE_MAP = [
```
상수 사용시 적용되는 컨벤션은 snake_case + 대문자입니다!

Reply:
넵 바로 적용하겠습니다!!